### PR TITLE
EZP-31719: Verbatim HTML in end of life information on Dashboard

### PR DESCRIPTION
**Pull request by glye**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-31719
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | TBD
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

The alert box for end-of-life shows verbatim (escaped) HTML. This adds `raw` operators and indents in the same way as similar texts in the same template.

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/views/themes/admin/ui/dashboard/block/ez.html.twig [hunk 1]
```diff
@@ -14,8 +14,11 @@
     {% if not ez.release %}
         {% set severity = 1 %}
         <div class="alert alert-warning mb-0 mt-3" role="alert">
-            {{ 'dashboard.ez_version.release_not_determined'|trans|desc("The system could not find your <code>composer.lock</code> file. It's needed to determine information about
-            your eZ install, and recommended to be kept on project development to make sure same package versions are used across all environments.")}}
+            {{ 'dashboard.ez_version.release_not_determined'|trans
+            |desc("The system could not find your <code>composer.lock</code> file. It's needed to determine information about
+                your eZ install, and recommended to be kept on project development to make sure same package versions are
```
Comment (DominikaK) [inline src/bundle/Resources/views/themes/admin/ui/dashboard/block/ez.html.twig:19]:
```suggestion
                your eZ installation, and recommended to be kept on project development to make sure the same package versions are
```

Comment (DominikaK) [inline src/bundle/Resources/views/themes/admin/ui/dashboard/block/ez.html.twig:19]: Or even better:

```suggestion
                your eZ installation. It is recommended to keep it during project development to make sure the same package versions are
```

Comment (glye) [reply src/bundle/Resources/views/themes/admin/ui/dashboard/block/ez.html.twig:19]: Gracias.

Comment (glye) [reply src/bundle/Resources/views/themes/admin/ui/dashboard/block/ez.html.twig]: I went with the other one.

**Comment by micszo** [comment]
@lserwatka @adamwojs @webhdx (since Glye is on vacation) we need this change for `1.5` also

**Pull request merged**